### PR TITLE
[qt5-webengine] fix jumbo build error due to ResolveColor() redefinition

### LINKED ISSUE #37448: [qt5-webengine] QtWebEngine jumbo build fails with redefinition of blink::ResolveColor(..) since port version 5.15.13-0 on Linux
### Operating system

Linux (AlmaLinux 9.3)

### Compiler

GCC (stock distro compiler version 11.4.1)

### Steps to reproduce the behavior

```
mkdir example
cd example
git clone https://github.com/microsoft/vcpkg.git
./vcpkg/bootstrap-vcpkg.sh
./vcpkg/vcpkg install qt5-webengine --triplet=x64-linux-dynamic
```

The error doesn't look like it is vcpkg specific, so I have also reported it upstream: [QTBUG-123328](https://bugreports.qt.io/browse/QTBUG-123328). Basically both `src/3rdparty/chromium/third_party/blink/renderer/core/layout/svg/svg_layout_tree_as_text.cc` and `src/3rdparty/chromium/third_party/blink/renderer/core/layout/svg/layout_svg_resource_paint_server.cc` define a static function `ResolveColor()` with the exact same signature causing this redefinition error when running a jumbo build (the default).

### Failure logs

Error from example/vcpkg/buildtrees/qt5-webengine/package-build-x64-linux-dynamic-dbg-out.log:
```
FAILED: obj/third_party/blink/renderer/core/layout/svg/svg_layout/svg_layout_jumbo_4.o
/usr/bin/c++ -MMD -MF obj/third_party/blink/renderer/core/layout/svg/svg_layout/svg_layout_jumbo_4.o.d -DUSE_UDEV -DUSE_AURA=1 -DUSE_NSS_CERTS=1 -DUSE_OZONE=1 -DTOOLKIT_QT -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D_DEBUG -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DBLINK_CORE_IMPLEMENTATION=1 -DGL_GLEXT_PROTOTYPES -DUSE_GLX -DUSE_EGL -DBLINK_IMPLEMENTATION=1 -DINSIDE_BLINK -DUSING_SYSTEM_ICU=1 -DICU_UTIL_DATA_IMPL=ICU_UTIL_DATA_STATIC -DUCHAR_TYPE=uint16_t -DWEBRTC_NON_STATIC_TRACE_EVENT_HANDLERS=0 -DWEBRTC_CHROMIUM_BUILD -DWEBRTC_POSIX -DWEBRTC_LINUX -DABSL_ALLOCATOR_NOTHROW=1 -DWEBRTC_USE_BUILTIN_ISAC_FIX=0 -DWEBRTC_USE_BUILTIN_ISAC_FLOAT=1 -DWEBRTC_USE_X11 -DWEBRTC_USE_X11 -DHAVE_SCTP -DNO_MAIN_THREAD_WRAPPING -DGOOGLE_PROTOBUF_NO_RTTI -DGOOGLE_PROTOBUF_NO_STATIC_INITIALIZER -DHAVE_PTHREAD -DSK_CODEC_DECODES_PNG -DSK_CODEC_DECODES_WEBP -DSK_ENCODE_PNG -DSK_ENCODE_WEBP -DSK_USER_CONFIG_HEADER=\"../../skia/config/SkUserConfig.h\" -DSK_GL -DSK_CODEC_DECODES_JPEG -DSK_ENCODE_JPEG -DSK_USE_LIBGIFCODEC -DSK_VULKAN_HEADER=\"../../skia/config/SkVulkanConfig.h\" -DSK_VULKAN=1 -DSK_SUPPORT_GPU=1 -DSK_GPU_WORKAROUNDS_HEADER=\"gpu/config/gpu_driver_bug_workaround_autogen.h\" -DV8_ENABLE_CHECKS -DV8_COMPRESS_POINTERS -DV8_31BIT_SMIS_ON_64BIT_ARCH -DV8_DEPRECATION_WARNINGS -DV8_REVERSE_JSARGS -DUSING_SYSTEM_ICU=1 -DICU_UTIL_DATA_IMPL=ICU_UTIL_DATA_STATIC -DUCHAR_TYPE=uint16_t -DCRASHPAD_ZLIB_SOURCE_EXTERNAL -DLEVELDB_PLATFORM_CHROMIUM=1 -DLEVELDB_PLATFORM_CHROMIUM=1 -DSUPPORT_WEBGL2_COMPUTE_CONTEXT=1 -DWTF_USE_WEBAUDIO_PFFFT=1 -DUSE_BLINK_V8_BINDING_NEW_IDL_INTERFACE -DUSE_SYSTEM_LIBJPEG -DV8_ENABLE_CHECKS -DV8_COMPRESS_POINTERS -DV8_31BIT_SMIS_ON_64BIT_ARCH -DV8_DEPRECATION_WARNINGS -DV8_REVERSE_JSARGS -DLIBXSLT_STATIC -I. -Igen -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/perfetto/include -Igen/third_party/perfetto/build_config -Igen/third_party/perfetto -Igen -Igen -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/libyuv/include -Igen -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/jsoncpp/source/include -Igen -Igen -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/vulkan_headers/include -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/khronos -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/gpu -Igen/third_party/dawn/src/include -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/dawn/src/include -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/abseil-cpp -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/boringssl/src/include -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/protobuf/src -Igen/protoc_out -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/ced/src -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/webrtc_overrides -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/webrtc -Igen/third_party/webrtc -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/protobuf/src -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/skia -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/libgifcodec -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/vulkan/include -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/vulkan_headers/include -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/angle/include -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/angle/src/common/third_party/base -Igen/angle -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/angle/include -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/v8/include -Igen/v8/include -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/crashpad/crashpad -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/crashpad/crashpad/compat/non_mac -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/crashpad/crashpad/compat/linux -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/crashpad/crashpad/compat/non_win -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/zlib -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/mesa_headers -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/libwebm/source -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/leveldatabase -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/leveldatabase/src -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/leveldatabase/src/include -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/iccjpeg -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/ots/include -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/v8/include -Igen/v8/include -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/libxml/src/include -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/libxml/linux/include -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/libxslt/src -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/snappy/src -I../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/snappy/linux -fno-strict-aliasing --param=ssp-buffer-size=4 -fstack-protector -funwind-tables -fPIC -pipe -pthread -m64 -Wno-builtin-macro-redefined -D__DATE__= -D__TIME__= -D__TIMESTAMP__= -Wall -Wno-unused-local-typedefs -Wno-maybe-uninitialized -Wno-deprecated-declarations -fno-delete-null-pointer-checks -Wno-comments -Wno-packed-not-aligned -Wno-dangling-else -Wno-missing-field-initializers -Wno-unused-parameter -fno-omit-frame-pointer -fvisibility=hidden -Os -g0 -Wno-unused-variable -I/home/ts/src/example/vcpkg/installed/x64-linux-dynamic/lib/pkgconfig/../../include -I/home/ts/src/example/vcpkg/installed/x64-linux-dynamic/lib/pkgconfig/../../include -I/usr/include/nss3 -I/usr/include/nspr4 -I/home/ts/src/example/vcpkg/installed/x64-linux-dynamic/lib/pkgconfig/../../include -I/home/ts/src/example/vcpkg/installed/x64-linux-dynamic/lib/pkgconfig/../../include/libpng16 -I/home/ts/src/example/vcpkg/installed/x64-linux-dynamic/lib/pkgconfig/../../include -I/home/ts/src/example/vcpkg/installed/x64-linux-dynamic/lib/pkgconfig/../../include -I/home/ts/src/example/vcpkg/installed/x64-linux-dynamic/lib/pkgconfig/../../include/webp -DLIBXML_STATIC= -std=gnu++14 -Wno-narrowing -Wno-class-memaccess -Wno-attributes -Wno-class-memaccess -Wno-subobject-linkage -Wno-invalid-offsetof -Wno-return-type -Wno-deprecated-copy -fno-exceptions -fno-rtti -fvisibility-inlines-hidden -Wno-deprecated-declarations -c gen/third_party/blink/renderer/core/layout/svg/svg_layout_jumbo_4.cc -o obj/third_party/blink/renderer/core/layout/svg/svg_layout/svg_layout_jumbo_4.o
In file included from gen/third_party/blink/renderer/core/layout/svg/svg_layout_jumbo_4.cc:16:
./../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/blink/renderer/core/layout/svg/svg_layout_tree_as_text.cc:255:30: error: redefinition of ‘base::Optional<blink::Color> blink::ResolveColor(const blink::ComputedStyle&, const blink::SVGPaint&, const blink::SVGPaint&)’
  255 | static base::Optional<Color> ResolveColor(const ComputedStyle& style,
      |                              ^~~~~~~~~~~~
In file included from gen/third_party/blink/renderer/core/layout/svg/svg_layout_jumbo_4.cc:5:
./../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/blink/renderer/core/layout/svg/layout_svg_resource_paint_server.cc:97:30: note: ‘base::Optional<blink::Color> blink::ResolveColor(const blink::ComputedStyle&, const blink::SVGPaint&, const blink::SVGPaint&)’ previously defined here
   97 | static base::Optional<Color> ResolveColor(const ComputedStyle& style,
      |                              ^~~~~~~~~~~~
In file included from gen/third_party/blink/renderer/core/layout/svg/svg_layout_jumbo_4.cc:16:
./../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/blink/renderer/core/layout/svg/svg_layout_tree_as_text.cc:255:30: warning: ‘base::Optional<blink::Color> blink::ResolveColor(const blink::ComputedStyle&, const blink::SVGPaint&, const blink::SVGPaint&)’ defined but not used [-Wunused-function]
  255 | static base::Optional<Color> ResolveColor(const ComputedStyle& style,
      |                              ^~~~~~~~~~~~
```

### Additional context

```
$ $ cat /etc/redhat-release
AlmaLinux release 9.3 (Shamrock Pampas Cat)

$ $ gcc --version
gcc (GCC) 11.4.1 20230605 (Red Hat 11.4.1-2)
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

$ rpm -qa |grep devel | sort
clang-devel-16.0.6-1.el9.x86_64
gcc-toolset-13-libquadmath-devel-13.1.1-4.2.el9.x86_64
gcc-toolset-13-libstdc++-devel-13.1.1-4.2.el9.x86_64
glibc-devel-2.34-83.el9_3.7.x86_64
gmp-devel-6.2.0-13.el9.x86_64
libdrm-devel-2.4.115-1.el9.x86_64
libfontenc-devel-1.1.3-17.el9.x86_64
libglvnd-core-devel-1.3.4-1.el9.x86_64
libglvnd-devel-1.3.4-1.el9.x86_64
libICE-devel-1.0.10-8.el9.x86_64
libomp-devel-16.0.6-1.el9.x86_64
libpciaccess-devel-0.16-6.el9.x86_64
libSM-devel-1.2.3-10.el9.x86_64
libstdc++-devel-11.4.1-2.1.el9.alma.x86_64
libuuid-devel-2.37.4-15.el9.x86_64
libX11-devel-1.7.0-8.el9.x86_64
libXau-devel-1.0.9-8.el9.x86_64
libXaw-devel-1.0.13-19.el9.x86_64
libxcb-devel-1.13.1-9.el9.x86_64
libXcomposite-devel-0.4.5-7.el9.x86_64
libxcrypt-devel-4.4.18-3.el9.x86_64
libXcursor-devel-1.2.0-7.el9.x86_64
libXdamage-devel-1.1.5-7.el9.x86_64
libXdmcp-devel-1.1.3-8.el9.x86_64
libXext-devel-1.3.4-8.el9.x86_64
libXfixes-devel-5.0.3-16.el9.x86_64
libXi-devel-1.7.10-8.el9.x86_64
libXinerama-devel-1.1.4-10.el9.x86_64
libxkbcommon-devel-1.0.3-4.el9.x86_64
libxkbcommon-x11-devel-1.0.3-4.el9.x86_64
libxkbfile-devel-1.1.0-8.el9.x86_64
libxml2-devel-2.9.13-5.el9_3.x86_64
libXmu-devel-1.1.3-8.el9.x86_64
libXpm-devel-3.5.13-8.el9_1.x86_64
libXrandr-devel-1.5.2-8.el9.x86_64
libXrender-devel-0.9.10-16.el9.x86_64
libXres-devel-1.2.0-14.el9.x86_64
libXScrnSaver-devel-1.2.3-10.el9.x86_64
libxshmfence-devel-1.3-10.el9.x86_64
libXt-devel-1.2.0-6.el9.x86_64
libXtst-devel-1.2.3-16.el9.x86_64
libXv-devel-1.0.11-16.el9.x86_64
libXxf86vm-devel-1.1.4-18.el9.x86_64
mesa-libEGL-devel-23.1.4-1.el9.x86_64
mesa-libGL-devel-23.1.4-1.el9.x86_64
mesa-libGLU-devel-9.0.1-6.el9.x86_64
mpfr-devel-4.1.0-7.el9.x86_64
nspr-devel-4.35.0-6.el9_3.x86_64
nss-devel-3.90.0-6.el9_3.x86_64
nss-softokn-devel-3.90.0-6.el9_3.x86_64
nss-softokn-freebl-devel-3.90.0-6.el9_3.x86_64
nss-util-devel-3.90.0-6.el9_3.x86_64
valgrind-devel-3.21.0-7.el9.x86_64
wayland-protocols-devel-1.31-1.el9.noarch
xcb-util-cursor-devel-0.1.3-13.el9.x86_64
xcb-util-devel-0.4.0-19.el9.x86_64
xcb-util-image-devel-0.4.0-19.el9.x86_64
xcb-util-keysyms-devel-0.4.0-17.el9.x86_64
xcb-util-renderutil-devel-0.3.9-20.el9.x86_64
xcb-util-wm-devel-0.4.1-22.el9.x86_64
xcb-util-xrm-devel-1.3-8.el9.x86_64
xkeyboard-config-devel-2.33-2.el9.noarch
xorg-x11-proto-devel-2022.2-1.el9.noarch
xorg-x11-xtrans-devel-1.4.0-8.el9.noarch
xz-devel-5.2.5-8.el9_0.x86_64
zlib-devel-1.2.11-40.el9.x86_64
```

### PR DESCRIPTION
**Pull request by tsondergaard**
Both of the following two files define a static function ResolveColor() with the exact same signature causing a redefinition error when running a jumbo build

* third_party/blink/renderer/core/layout/svg/svg_layout_tree_as_text.cc
* third_party/blink/renderer/core/layout/svg/layout_svg_resource_paint_server.cc

Error reported:

```
    In file included from gen/third_party/blink/renderer/core/layout/svg/svg_layout_jumbo_4.cc:16:
    ./../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/blink/renderer/core/layout/svg/svg_layout_tree_as_text.cc:255:30: error: redefinition of 'base::Optional<blink::Color> blink::ResolveColor(const blink::ComputedStyle&, const blink::SVGPaint&, const blink::SVGPaint&)'
      255 | static base::Optional<Color> ResolveColor(const ComputedStyle& style,
          |                              ^~~~~~~~~~~~
    In file included from gen/third_party/blink/renderer/core/layout/svg/svg_layout_jumbo_4.cc:5:
    ./../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/blink/renderer/core/layout/svg/layout_svg_resource_paint_server.cc:97:30: note: 'base::Optional<blink::Color> blink::ResolveColor(const blink::ComputedStyle&, const blink::SVGPaint&, const blink::SVGPaint&)' previously defined here
       97 | static base::Optional<Color> ResolveColor(const ComputedStyle& style,
         |                              ^~~~~~~~~~~~
    In file included from gen/third_party/blink/renderer/core/layout/svg/svg_layout_jumbo_4.cc:16:
    ./../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/blink/renderer/core/layout/svg/svg_layout_tree_as_text.cc:255:30: warning: 'base::Optional<blink::Color> blink::ResolveColor(const blink::ComputedStyle&, const blink::SVGPaint&, const blink::SVGPaint&)' defined but not used [-Wunused-function]
      255 | static base::Optional<Color> ResolveColor(const ComputedStyle& style,
```

Fixes #37448

See also QTBUG-123328

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [x] The "supports" clause reflects platforms that may be fixed by this new version.
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/qt5-webengine/0001-Fix-jumbo-build-error-due-to-ResolveColor-redefiniti.patch [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    ./../../../../src/5.15.13-38459bd7fb.clean/src/3rdparty/chromium/third_party/blink/renderer/core/layout/svg/svg_layout_tree_as_text.cc:255:30: warning: 'base::Optional<blink::Color> blink::ResolveColor(const blink::ComputedStyle&, const blink::SVGPaint&, const blink::SVGPaint&)' defined but not used [-Wunused-function]
+      255 | static base::Optional<Color> ResolveColor(const ComputedStyle& style,
+
+Vcpkg issue: https://github.com/microsoft/vcpkg/issues/37448
+
+Task-number: QTBUG-123328
+---
+ .../core/layout/svg/layout_svg_resource_paint_server.cc       | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
```
Comment (JonLiu1993) [inline ports/qt5-webengine/0001-Fix-jumbo-build-error-due-to-ResolveColor-redefiniti.patch:33]:
```suggestion

```

Comment (JonLiu1993) [reply ports/qt5-webengine/0001-Fix-jumbo-build-error-due-to-ResolveColor-redefiniti.patch:33]: @tsondergaard, Thanks for your PR, We recommend that patches be optimized to the minimum, so I recommend that we only keep the core modifications.

**Comment by JonLiu1993** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". That way, I can be aware that you've responded since you can't modify the tags.

**Review by JonLiu1993 — DISMISSED** [review]

**Comment by BillyONeal** [comment]
Why does this build problem affecting jumbo builds affect vcpkg, which does not do jumbo builds?

**Comment by tsondergaard** [comment]
@BillyONeal, I don't see anywhere in ports/qt5-webengine that explicitly disables jumbo builds. This is from config-x64-linux-dynamic-dbg-err.log on my AlmaLinux 9.3:

```
Checking for xproto (glproto)... yes
Checking for xtst... yes
Checking for xkbfile... yes
Checking for jumbo build merge limit... 8
Checking for linker supports -z noexecstack... yes
Checking for freetype >= 2.4.2... yes
Checking for glib-2.0 >= 2.32.0... no
Checking for harfbuzz >= 2.4.0... yes
Checking for icu >= 65... yes
Checking for compatible jpeglib... yes
Checking for lcms2... no
Checking for libevent... no
Checking for libvpx... no
Checking for libwebp, libwebpmux and libwebpdemux... yes
Checking for compatible libxml2 and libxslt... no
Checking for minizip... no
Checking for system ninja... yes
Checking for opus... no
Checking for libpng >= 1.6.0... yes
Checking for re2... no
Checking for snappy... no
Checking for zlib... yes
Checking for alsa... no
Checking for embedded build... no
Checking for poppler-cpp... no
Checking for pulseaudio >= 0.9.10... no
Done running configuration tests.

Configure summary:

Qt WebEngine Build Tools:
  Use System Ninja ....................... yes
  Jumbo Build Merge Limit ................ 8
  Developer build ........................ no
```

The jumbo-build of QtWebEngine, or Chromium really, can be explicitly turned off, but that is just going to make it build a lot slower and it is already painfully slow.

**Comment by dg0yt** [comment]
> Why does this build problem affecting jumbo builds affect vcpkg, which does not do jumbo builds?

qt5-webenengine embeds a chromium build. That's the root of all difficulties with this port.

**Comment by tsondergaard** [comment]
One more thing to point out - I believe that the jumbo merge limit has an impact on whether the compile will fail or not. The jumbo build creates these bunches of sources and compiles them in a single compilation unit. If third_party/blink/renderer/core/layout/svg/svg_layout_tree_as_text.cc and third_party/blink/renderer/core/layout/svg/layout_svg_resource_paint_server.cc end up being compiled in the same compilation unit then you will run into this bug.

**Comment by BillyONeal** [comment]
> I don't see anything in ports/qt5-webengine that explicitly disables jumbo builds.

Jumbo builds are normally an opt *in* thing rather than opt *out* :sigh: .

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks :)

**Pull request merged**